x-pack/plugins/ml/server/routes/schemas/alerting_schema.ts

```diff
@@ -58,3 +60,47 @@ export type MlAnomalyDetectionAlertParams = TypeOf<typeof mlAnomalyDetectionAler
 export type MlAnomalyDetectionAlertPreviewRequest = TypeOf<
   typeof mlAnomalyDetectionAlertPreviewRequest
 >;
+
+export const anomalyDetectionJobsHealthRuleParams = schema.object({
+  includeJobs: jobsSelectionSchema,
+  excludeJobs: schema.nullable(jobsSelectionSchema),
+  testsConfig: schema.nullable(
+    schema.object({
+      dataFeed: schema.nullable(
```

Comment:
Can this be `datafeed`?

Reply:
Renamed in https://github.com/elastic/kibana/pull/106084/commits/b08031efbc265847be7d3617ca90253110b74fc9